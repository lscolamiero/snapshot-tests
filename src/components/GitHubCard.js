import Card from 'react-bootstrap/Card';
import githubimg from '../githubimg.jpeg'

function GithubCard() {
    return (
      <Card style={{ width: '18rem' }}>
        <Card.Img variant="top" src={githubimg} />
        <Card.Body>
          <Card.Title>lscolamiero</Card.Title>
          <Card.Text>
            We are young developers with a great future.
          </Card.Text>
        </Card.Body>
      </Card>
    );
  }
  
  export default GithubCard;