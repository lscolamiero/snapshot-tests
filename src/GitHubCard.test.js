import { render, screen } from '@testing-library/react';
import GitHubCard from './components/GitHubCard';
import renderer from 'react-test-renderer';

test('renders a snapshot of my github account', () => {
    const tree = renderer.create(<GitHubCard/>).toJSON()
    console.log(tree)
    expect(tree).toMatchSnapshot()
  })