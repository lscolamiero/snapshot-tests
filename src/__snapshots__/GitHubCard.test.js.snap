// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of my github account 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="githubimg.jpeg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      lscolamiero
    </div>
    <p
      className="card-text"
    >
      We are young developers with a great future.
    </p>
  </div>
</div>
`;
